src/components/PageBlocks/CommentBase.vue

```diff
@@ -0,0 +1,339 @@
+            />
+            <i
+              v-else
+              class="text-secondary h6"
+            >{{ $t('field.noPermission') }}</i>
+            <span class="ml-auto text-muted">
+              {{ new Date(record.updatedAt || record.createdAt).toLocaleString("en-US") }}
+            </span>
+          </div>
+          <div class="border p-3 d-flex flex-column">
+            <field-viewer
+              v-if="titleField.canReadRecordValue"
+              class="mb-3 text-muted font-weight-bold"
+              :field="titleField"
+              :record="record"
+              :namespace="namespace"
+              value-only
+            />
+            <i
+              v-else
+              class="text-secondary h6"
+            >{{ $t('field.noPermission') }}</i>
+            <field-viewer
+              v-if="contentField.canReadRecordValue"
+              :field="contentField"
+              :record="record"
+              :namespace="namespace"
+              value-only
+            />
+            <i
+              v-else
+              class="text-secondary h6"
+            >{{ $t('field.noPermission') }}</i>
+          </div>
+        </b-list-group-item>
+      </b-list-group>
+    </section>
+  </wrap>
+</template>
+<script>
+import { mapGetters } from 'vuex'
+import base from './base'
+import FieldViewer from 'corteza-webapp-compose/src/components/ModuleFields/Viewer'
+import users from 'corteza-webapp-compose/src/mixins/users'
+import { compose } from '@cortezaproject/corteza-js'
+
+export default {
+  components: {
+    FieldViewer,
+  },
+
+  extends: base,
+
+  mixins: [
+    users,
+  ],
+
+  data () {
+    return {
+      filter: {
+        sort: '',
+        filter: '',
+      },
+
+      records: [],
+      newRecord: {
+        title: '',
+        content: '',
+      },
+      sortAsc: false,
+
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      getModuleByID: 'module/getByID',
+      pages: 'page/set',
+    }),
+
+    roModule () {
+      return this.getModuleByID(this.moduleID)
+    },
+
+    roRecordPage () {
+      return this.pages.find(p => p.moduleID === this.moduleID)
+    },
+
+    moduleID () {
+      return this.options.moduleID
+    },
+
+    titleField () {
+      const { titleField } = this.options
+
+      if (!titleField) {
+        return undefined
+      }
+
+      return this.roModule.fields.find(f => f.name === titleField) || {}
+    },
+
+    contentField () {
+      const { contentField } = this.options
+
+      if (!contentField) {
+        return undefined
+      }
+
+      return this.roModule.fields.find(f => f.name === contentField) || {}
+    },
+
+    authorField () {
+      const { authorField } = this.options
+      if (!authorField) {
+        return undefined
+      }
+
+      return this.roModule.fields.find(f => f.name === authorField) || {}
+    },
+
+    canAddRecord () {
+      return this.roModule && this.roModule.canCreateRecord && this.roRecordPage
```

Comment:
why is `roRecordPage` in the condition here? There is no need to have a record page to add comment record.